misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -398,4 +402,25 @@ class FakeRedis : Redis {
   override fun publish(channel: String, message: String) {
     throw NotImplementedError("Fake client not implemented for this operation")
   }
+
+  override fun zadd(
+    key: String,
+    score: Double,
+    member: ByteString,
+    options: Set<String>
+  ): Long {
+    throw NotImplementedError("Fake client not implemented for this operation")
+  }
+
+  override fun zadd(
+    key: String,
+    scoreMembers: Map<ByteString, Double>,
+    options: Set<String>,
+  ): Long {
+    throw NotImplementedError("Fake client not implemented for this operation")
+  }
+
+  override fun zscore(key: String, member: String): Double? {
+    throw NotImplementedError("Fake client not implemented for this operation")
```

Comment:
Can you implement these? You won't be able to write tests without these in most Misk codebases, because folks generally don't have Redis set up in Docker

Reply:
Hmm.... what's the difference between this FakeRedis and the textFixtures FakeRedis? Looks like there's been some changes to this codebase since I last work on it

Reply:
There is class level doc which says.
```
/** Mimics a Redis instance for testing. */
@Deprecated(
  message = "Moved to misk-redis-testing. This Fake does not emulate redis correctly.",
  replaceWith = ReplaceWith("misk.redis.testing.FakeRedis")
) 
class FakeRedis : Redis {
```
In the FakeRedisTests, the `misk.redis.testing.FakeRedis` gets used. I can still add a note in the error to direct folks to the other FakeRedis.

Reply:
Oh great. I missed this. Feel free to ignore my comments then